### PR TITLE
Python -> Python3

### PR DESCRIPTION
**Pull request by afuller-TT**
### Ticket
#14393

### Problem description
The scripts do not execute inside a clean Docker container because they call `python` instead of `python3`.

### What's changed
python -> python3

### Checklist
- [x] Post commit CI passes https://github.com/tenstorrent/tt-metal/actions/runs/12360690672
- [x] Device perf regressions passes https://github.com/tenstorrent/tt-metal/actions/runs/12358294738

### CODE REVIEW

**Review by mo-tenstorrent — APPROVED** [review]

File: .github/workflows/perf-device-models-impl.yaml [hunk 1]
```diff
@@ -22,22 +27,20 @@ jobs:
       LD_LIBRARY_PATH: ${{ github.workspace }}/build/lib
     runs-on: ${{ matrix.test-info.runs-on }}
     steps:
-      - uses: tenstorrent/tt-metal/.github/actions/checkout-with-submodule-lfs@main
       - name: Ensure weka mount is active
         run: |
           sudo systemctl restart mnt-MLPerf.mount
           sudo /etc/rc.local
           ls -al /mnt/MLPerf/bit_error_tests
+      - uses: tenstorrent/tt-metal/.github/actions/checkout-with-submodule-lfs@main
```
[the remark this answers is not in the document]
Comment (afuller-TT) [reply .github/workflows/perf-device-models-impl.yaml:35]: For now keeping the same checkout rules.
I *think* we're done with LFS in this repo.  But I'll leave that for a separate sweep across workflows.

**Review by esmalTT — APPROVED** [review]

File: .github/workflows/perf-device-models-impl.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (blozano-tt) [reply .github/workflows/perf-device-models-impl.yaml:35]: Some submodules still have LFS. It’s a goal to remove all of it in 2025.

[the remark this answers is not in the document]
Comment (tt-rkim) [reply .github/workflows/perf-device-models-impl.yaml:35]: Do we know which?

[the remark this answers is not in the document]
Comment (blozano-tt) [reply .github/workflows/perf-device-models-impl.yaml:35]: The “lfs” submodule is the one I recall. Could be more.

https://github.com/tenstorrent-metal/lfs

**Pull request merged**